synapse/handlers/events.py

```diff
@@ -61,6 +61,11 @@ def get_stream(self, auth_user_id, pagin_config, timeout=0,
         If `only_keys` is not None, events from keys will be sent down.
         """
 
+        if room_id:
+            blocked = yield self.store.is_room_blocked(room_id)
+            if blocked:
```

Comment:
fyi: you know you can write `if (yield self.store.is_room_blocked(room_id)):` ?

Reply:
Oh, yeah, I always forget that